src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,16 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const todoInfoClass = cn({ 'TodoInfo--completed': todo.completed });
+
+  return (
+    <article className={`TodoInfo ${todoInfoClass}`}>
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+
+      {todo.user ? <UserInfo user={todo.user} /> : null}
```

Comment:
don't need to ternary operator if you want to display only one block of code 
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```